pkg/lib/cockpit.js

```diff
@@ -4190,10 +4190,10 @@ function factory() {
         self.get = function get(path, params, headers) {
             return self.request({
                 method: "GET",
-                params: params,
-                path: path,
+                params,
+                path,
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-173310-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)